src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculatorTest.kt

```diff
@@ -752,32 +674,6 @@ internal class ProtectLevelCalculatorTest {
       assertThat(result.points).isEqualTo(2)
     }
 
-    @Test
```

Comment:
No inactive registrations any more

Reply:
nitpick: is it worth keeping belt and braches?